packages/hint/src/lib/cli/analyze.ts

```diff
@@ -448,16 +449,16 @@ export default async (actions: CLIOptions): Promise<boolean> => {
             updateCallback: undefined
         };
 
-        /* istanbul ignore else */
-        if (!actions.debug) {
+        // /* istanbul ignore else */
```

Comment:
Do we need this to be a single line comment or is a leftover from tests?

Reply:
It looks like a leftover from tests

Reply:
Fixed!